content/en/blog/_posts/2023-04-06-kubernetes-patch-releases-supported-go-versions.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: blog
+title: "Kubernetes Patch Releases: Staying On Supported Go Versions"
```

Comment:
I feel like this is burying the lede somewhat by not emphasizing the reason to stay on supported Go versions (namely security)

Keeping Kubernetes Secure with Updated Go Versions

?

Reply:
fair point, I like that better